R/MsBackendMzR-functions.R

```diff
@@ -37,6 +37,7 @@ MsBackendMzR <- function() {
         hdr$isolationWindowUpperOffset
     hdr$isolationWindowUpperOffset <- NULL
     hdr$isolationWindowLowerOffset <- NULL
+    ## Remove core spectra variables that contain only `NA`
```

Comment:
You want to do this in the future, or (#67)?

Reply:
I have it already in a local branch - just didn't want to push that to this pull request to not overcomplicate it (and to not push additional features to an already created PR).